<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>audio</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>bit rate</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>sample rate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>file format</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="audioBitRate"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="audioSampleRate"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="audioFileFormat"/>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="mux">
       <property name="text">
        <string>mux after record</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>video</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>148</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>bit rate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="videoBitRate"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Quality</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="videoQuality"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="videoProfile"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="videoLevel"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Profiles</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Levels</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QCheckBox" name="audioActive">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>audio is active</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>audioBitRate</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>setAudioBitRate(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>audioSampleRate</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>setAudioSampleRate(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>audioFileFormat</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>setAudioFileFormat(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mux</sender>
   <signal>clicked(bool)</signal>
   <receiver>Form</receiver>
   <slot>doMux(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>videoBitRate</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>setVideoBitRate(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>videoQuality</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>setVideoQuality(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>audioActive</sender>
   <signal>clicked(bool)</signal>
   <receiver>Form</receiver>
   <slot>isAudioActive(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>videoProfile</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>setVideoProfile(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>videoLevel</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>setVideoLevel(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setAudioBitRate(int)</slot>
  <slot>setAudioSampleRate(int)</slot>
  <slot>setAudioFileFormat(int)</slot>
  <slot>doMux(bool)</slot>
  <slot>setVideoBitRate(int)</slot>
  <slot>setVideoQuality(int)</slot>
  <slot>isAudioActive(bool)</slot>
  <slot>setVideoProfile(int)</slot>
  <slot>setVideoLevel(int)</slot>
 </slots>
</ui>
